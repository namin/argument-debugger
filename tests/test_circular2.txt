Democracy is the best form of government because it represents the will of the people.
We know it represents the will of the people because democratic elections show what people want.
Democratic elections show what people want because democracy is the best system for expressing preferences.