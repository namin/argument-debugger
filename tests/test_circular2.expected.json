{
  "must_find": [],
  "may_find": ["circular", "unsupported_premise"],
  "must_not_find": ["false_dichotomy"]
}